app/assets/stylesheets/application.scss

```diff
@@ -13,4 +13,20 @@
  *= require_tree .
  *= require_self
  */
- @import "bootstrap";
+@import "bootstrap";
+/* Base */
+body {
+  padding: 0;
+  margin: 0;
+  h1 {
+    color: hsl(0, 0%, 28%);
+    font-size: 25px;
+    font-weight: bold;
+  }
+  .hover-span:hover {
```

Comment:
The class name 'hover-span' is not descriptive. Consider using a more meaningful name.